.coveragerc

```diff
@@ -10,13 +10,5 @@ exclude_lines =
     # Don't complain if non-runnable code isn't run
     if __name__ == .__main__.:
 
-
-# Paths to omit from consideration
-omit =
-    # __main__.py exists only as a very basic wrapper around warehouse.cli
-    #   and exists only to provide setuptools and python -m a place to point
-    #   at.
-    */twine/__main__.py
-
```

Comment:
I was initially missing a test of the fallback in `_format_http_exception`, but was surprised that Codecov didn't complain about a drop in coverage. Then I found this. 😉 

Since `__main__.py` is doing non-trivial work, and already had tests, I think it should be considered during coverage.